permissions/src/androidMain/kotlin/dev/icerock/moko/permissions/PermissionsControllerImpl.kt

```diff
@@ -192,6 +193,22 @@ class PermissionsControllerImpl(
             listOf(Manifest.permission.ACCESS_FINE_LOCATION)
         }
 
+    private fun backgroundLocationCompat() =
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+            listOf(
+                Manifest.permission.ACCESS_FINE_LOCATION,
+                Manifest.permission.ACCESS_COARSE_LOCATION,
+                Manifest.permission.ACCESS_BACKGROUND_LOCATION,
```

Comment:
why we need more permissions on S that on Q?

Reply:
from Q(29) we ask for [ACCESS_BACKGROUND_LOCATION ](https://developer.android.com/reference/android/Manifest.permission#ACCESS_BACKGROUND_LOCATION) and with it we must also request either [ACCESS_COARSE_LOCATION](https://developer.android.com/reference/android/Manifest.permission#ACCESS_COARSE_LOCATION) or [ACCESS_FINE_LOCATION](https://developer.android.com/reference/android/Manifest.permission#ACCESS_FINE_LOCATION). Requesting this permission by itself doesn't give you location access.

from S(31) we ask [both](https://developer.android.com/develop/sensors-and-location/location/permissions#approximate-request) [ACCESS_COARSE_LOCATION](https://developer.android.com/reference/android/Manifest.permission#ACCESS_COARSE_LOCATION) and [ACCESS_FINE_LOCATION](https://developer.android.com/reference/android/Manifest.permission#ACCESS_FINE_LOCATION). like in `fineLocationCompat()` method